grease-cli/src/Grease/Main.hs

```diff
@@ -269,6 +273,33 @@ withMemOptions opts k =
           }
    in k
 
+loadDwarfpreconditions ::
+  ExtShape ext ~ PtrShape ext w =>
+  Mem.HasPtrWidth w =>
+  Symbolic.SymArchConstraints arch =>
+  -- | Is DWARF enabled
+  Bool ->
```

Comment:
If I understand correctly, then if this is `False`, then this will basically just become the identity function, which feels a bit strange design-wise. I would prefer to have this function unconditionally load DWARF preconditions, and then have callers decide whether to apply the function or not...

Reply:
Fair enough, I didn't realize that there was prior art. I'm content with copying what `loadInitialPreconditions` does.